packages/flutter/test/material/list_tile_test.dart

```diff
@@ -1076,8 +1077,18 @@ void main() {
     final TestGesture gesture = await tester.startGesture(
       tester.getRect(find.byType(ListTile)).center,
     );
+    await tester.pump();
     await tester.pump(const Duration(milliseconds: 200));
-    expect(find.byType(Material), paints..circle(x: 50, y: 50, color: const Color(0xff88ff88)));
+    expect(
+      find.byType(Material),
+      paints
+        ..rect()
+        ..rect()
```

Comment:
On Web, this second `..rect()` should be a `..circle()` (Ripple effect)

Reply:
Do you recommend splitting tests into web and mobile? or disable web?

Reply:
I would not disable web but not strong opinion on this.
Maybe two expect clauses with a comment before the web one. That way when InkSparkle will be activated on Web, it will be easy to remove this second expect clause.